### PR TITLE
Updated export of icons from Figma

### PR DESCRIPTION
**Pull request by hashibot-hds**
### :pushpin: Summary

Adding an updated category list to the icon library for the Helios website. In the JSON package, "Category" will accompany the new category name for all icons. 

### :link: External links

Jira ticket: [HDS-2094](https://hashicorp.atlassian.net/browse/HDS-2094)
Figma file: [Flight Library](https://www.figma.com/file/TLnoT5AYQfy3tZ0H68BgOr/Flight-Icons?type=design&node-id=164%3A0&mode=design&t=cp5uSiWNOgYYJqt4-1)

***

### 👀 Component checklist

- [x] A changelog entry was added via [Changesets](https://github.com/changesets/changesets) if needed ([instructions here](https://github.com/hashicorp/design-system/blob/main/packages/flight-icons/CONTRIBUTING.md#updating-the-icons-in-flight-icons) / [expected format here](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#new-icons))
- [x] Percy was checked for any visual regression

[HDS-2094]: https://hashicorp.atlassian.net/browse/HDS-2094?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/beige-cups-raise.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+This PR introduces upates to the category names in the Flight library
```
Comment (heatherlarsen) [inline .changeset/beige-cups-raise.md:5]: We don't use language like "This PR introduces..." in the changesets. What about something more descriptive, like: 

> Reorganized the icons within the "Core" icon set into 16 more descriptive categories.

Note our Changelog content best practices: https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#content-formatting

**Review by didoo — APPROVED** [review]
👌

File: .changeset/beige-cups-raise.md (hunk 1 above)
Comment (andgen404) [inline .changeset/beige-cups-raise.md:5]:
```suggestion
Reorganized the icons within the "Core" icon set into 16 more descriptive categories.```

File: .changeset/beige-cups-raise.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Reorganized the icons within the "Core" icon set into 16 more descriptive categories.```
```
Comment (heatherlarsen) [inline .changeset/beige-cups-raise.md:5]: small typo

```suggestion
Reorganized the icons within the "Core" icon set into 16 more descriptive categories.
```

**Review by heatherlarsen — APPROVED** [review]

**Pull request merged**